src/WebJobs.Script/runtimeassemblies.json

```diff
@@ -479,6 +479,18 @@
       "name": "Microsoft.Azure.AppService.Proxy.Runtime",
       "resolutionPolicy": "private"
     },
+    {
+      "name": "Microsoft.Azure.Cosmos.Client",
```

Comment:
weird that these need to be added since you aren't touching any code in `/src`

Reply:
Without this we get a `MethodMissing` exception - this PR doesn't touch anything in `/src` but the test project references the projects there and something about the way the test host is built and pulls in the Cosmos packages is causing issues.